hooks/post/10-post_install.rb

```diff
@@ -61,7 +61,7 @@ def new_install?
   Once this is completed run the steps below to start the smartproxy installation:
 
   1. Ensure that the foreman-installer-katello package is installed on the system.
-  2. Copy <%= color("#{certs_tar}", :info) %> to the system <%= color("#{capsule_fqdn}", :info) %>
+  2. Copy the following file <%= color("#{certs_tar}", :info) %> to the system <%= color("#{capsule_fqdn}", :info) %> at the following location <%= color("#{certs_tar}", :info) %>
```

Comment:
It looks better, but the problem is the user could run:

```
capsule-certs-generate --capsule-fqdn "foo.example.com" --certs-tar "certs.tar.gz"
 ```

Or worse:

```
capsule-certs-generate --capsule-fqdn "foo.example.com" --certs-tar "/home/joe/certs.tar.gz"
 ```

In both cases certs_tar will just have exactly what the user gave us, so it's not helping in suggesting a place when running capsule-certs-generate.

I think you should do this:

```
2. Copy the following file <%= color("#{certs_tar}", :info) %> to the system <%= color("#{capsule_fqdn}", :info) %> at the following location <%= color("#{File.join("/root", File.basename(certs_tar))}", :info) %> 
```

The first certs_tar is what the user gave us, but the suggested copy location would end up being /root/certs.tar.gz as the suggested destination.